nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/validation/DocumentValidationException.java

```diff
@@ -38,10 +43,21 @@
     private DocumentValidationReport report;
 
     public DocumentValidationException(DocumentValidationReport report) {
-        super();
+        super(HttpServletResponse.SC_PRECONDITION_FAILED);
```

Comment:
Why changing `BAD_REQUEST` (400) to `PRECONDITION_FAILED` (412) ? It seems the second one is not adapted to our case, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412. It seems more adapted in cases such as `changeToken` has changed.

I think we're more in a `400` case, but if you want to distinguish this validation error leveraging the http status code, `422` seems more appropriate to what we do. Wdyt ?

Reply:
Indeed, there's a pending debate, I was hesitate with `422`

Reply:
Let's go for `422` then ;)